apps/data/database/initdb.sql

```diff
@@ -0,0 +1,91 @@
+CREATE FUNCTION trigger_update_modified()
```

Comment:
This shouldn't be in this PR. If it's waiting on the DB changes then it should be on a branch off of the DB set up branch, doing it like this leads to conflicts if the DB branch is changed

Reply:
Same goes for all the other DB setup related files

Reply:
I branched off the db setup, but I accidently made the PR to merge into master. my bad.